UndertaleModCli/CommandOptions/DumpOptions.cs

```diff
@@ -2,6 +2,9 @@
 
 namespace UndertaleModCli;
 
+#nullable enable
```

Comment:
Please don't. Either actually fix the warnings, or keep them as is. Silently repressing them just hides that there's an issue present.

Reply:
There's already a ReSharper disable for this, which specifies that the properties are applied via reflection. I'm not sure what the alternative to this is.

Reply:
That may be true for some of these but not all of them. Would it not be better to enable <Nullable> in the project file and add ? to the types when necessary, since that's the default in .NET 8 anyway?

Reply:
I'm considering (for now) just disabling and removing nullables from the CLI project altogether. Fully adding nullables is a bit of a task, especially since it interfaces with UndertaleModLib, which does not have them enabled (although it should still work). Going through that (and possibly also on the library/GUI) can be for later PRs. Any thoughts?

Reply:
So you would remove them now to add them later? Bit of double work but that's ok if doing all the null checks is too much for now.

Reply:
I'm not going to commit myself (or anyone else) to adding them later, at least for now. But yes, I'm considering removing them now so that, if we really want to commit to nullables in this project (the CLI one specifically, or UTMT as a whole), we can do so in later PRs.

Reply:
Removed the nullables for now. On a second look, it seems like nullables aren't being taken advantage of properly in the first place, anyway, so this seems like the best choice. Future PRs can add nullables project-wide if we want to do that.